packages/repository-json-schema/src/filter-json-schema.ts

```diff
@@ -14,6 +14,11 @@ export interface FilterSchemaOptions {
    *
    */
   setTitle?: boolean;
+
+  /**
+   * To exclude one or more property from `filter`
+   */
+  exclude?: string[] | string;
```

Comment:
:+1:

I see that both `string[]` and `string` flavors are supported, that's great ergonomics! Could you please double check that we have test coverage for both of them? When I was skimming through the tests, I saw `string[]` variant only.

This is important because the CLI templates are using `string` flavor, we should verify that such flavor works as intended!